src/workspace-data/Data.js

```diff
@@ -214,7 +214,7 @@ const DataTypeSection = ({ title, error, retryFunction, children }) => {
         ),
     },
     [
-      !!children?.length &&
+      children &&
```

Comment:
I don't believe this is equivalent:
<img width="499" alt="Screenshot 2025-03-05 at 11 26 15 AM" src="https://github.com/user-attachments/assets/613039a9-9821-4a64-8971-207ea9ec9e1a" />

I also don't think it changes anything in practice when rendering the page - your call on whether to keep this change or not

Reply:
This check prevented the `Workspace Data` button from showing up since the `DataTypeSection` Component needs the `children` to be an array and when I remove the banner the children for `Other Data` its children become an object hence why I changed the check if any children are present!